modules/google_service_account/main.tf

```diff
@@ -40,6 +40,15 @@ resource "google_project_iam_member" "in_project_roles" {
   project = var.project
   role    = each.value
   member  = "serviceAccount:${google_service_account.service_account.email}"
+  dynamic "condition" {
+    for_each = var.in_project_condition_expression != null && var.in_project_condition_title != null ? [1] : []
```

Comment:
You can change the default to an empty array to clean up this code.  Same for the other instances of this below.